contents/newsletter/think-like-a-growth-engineer.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to think like a growth engineer
+date: 2024-09-16
+author:
+ - ian-vanagas
+featuredImage: >-
+featuredImageType: full
+tags:
+  - Engineering
+crosspost:
+  - Product engineers
+  - Founders
+  - Blog
+---
+
+From the outside, it can seem like growth engineers bounce around the product aimlessly: signup flows, product onboarding, revenue reporting, and subscription tracking. You might think all this doesn't matter, but as many growth teams show, incremental improvements can lead to massive gains for the entire business.
```

Comment:
```suggestion
From the outside, it can seem as if growth engineers bounce around the product aimlessly – tweaking signup and onboarding flows, and running endless experiments, many of which fail.

It's tempting to dismiss this as busywork, but that would be a mistake. Nearly every successful company has a thriving growth team at its core, constantly driving incremental improvements that accumulate to massive gains over time.
```

Reply:
Added an en dash because I know you love them ;)